dodo.py

```diff
@@ -1,107 +1,143 @@
-import functools
+import itertools
+import os
 import pathlib
+import shlex
 
-from doit.action import CmdAction as _CmdAction
+from doit.action import CmdAction
 
-CmdAction = functools.partial(_CmdAction, shell=False)
 
 HERE = pathlib.Path(__file__).parent
+PACKAGE_NAME = "light_the_torch"
 
+CI = os.environ.get("CI") == "1"
 
-def task_setup(pip="pip"):
-    return dict(
+DOIT_CONFIG = dict(
+    verbosity=2,
+    backend="json",
+)
+
+
+def do(*cmd):
+    if len(cmd) == 1:
+        cmd = cmd[0]
+    if isinstance(cmd, str):
+        cmd = shlex.split(cmd)
+    return CmdAction(cmd, shell=False, cwd=HERE)
+
+
+def task_install(pip="python -m pip"):
+    """Installs all development requirements and light-the-torch in development mode"""
+    yield dict(
+        name="requirements",
+        actions=[do(f"{pip} install -r requirements-dev.txt")],
+    )
+    yield dict(
+        name="install",
         actions=[
-            [pip, "install", "-r", HERE / "requirements-dev.txt"],
-            [pip, "install", "-e", HERE],
+            do(f"{pip} install -e ."),
```

Comment:
is `task_dep = ["install:requirements"]`?

Reply:
Nope. I probably should rename the task above to `dev-requirements`, since it does not install any runtime requirements. But thanks for flagging this inconsistency.